google-cloud-datastore/pom.xml

```diff
@@ -111,6 +115,19 @@
       <artifactId>jsr305</artifactId>
     </dependency>
 
+    <!-- OpenTelemetry -->
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>1.37.0</version>
```

Comment:
nit - can we pull the version from the managed shared dependencies?

Reply:
I did set it up as initially. But hit some dependency issue w.r.t. version. Let me retry.